packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/useTeachingPopoverBody.ts

```diff
@@ -1,34 +1,41 @@
 import * as React from 'react';
-import { getIntrinsicElementProps, slot } from '@fluentui/react-utilities';
+import { getIntrinsicElementProps, resolveShorthand } from '@fluentui/react-utilities';
 import type { TeachingPopoverBodyProps, TeachingPopoverBodyState } from './TeachingPopoverBody.types';
-
 /**
- * Create the state required to render TeachingPopoverBody.
- *
- * The returned state can be modified with hooks such as useTeachingPopoverBodyStyles_unstable,
- * before being passed to renderTeachingPopoverBody_unstable.
- *
- * @param props - props from this instance of TeachingPopoverBody
- * @param ref - reference to root HTMLDivElement of TeachingPopoverBody
+ * Returns the props and state required to render the component
+ * @param props - TeachingPopoverBody properties
+ * @param ref - reference to root HTMLElement of TeachingPopoverBody
  */
 export const useTeachingPopoverBody_unstable = (
   props: TeachingPopoverBodyProps,
-  ref: React.Ref<HTMLDivElement>,
+  ref: React.Ref<HTMLElement>,
 ): TeachingPopoverBodyState => {
+  const { as, media, mediaLength } = props;
+
+  let mediaShorthand;
+
+  // Image shouldn't be rendered if its src is not set
+  if (media?.src) {
+    mediaShorthand = resolveShorthand(media, {
+      required: false,
+      defaultProps: {
+        alt: '',
+        role: 'presentation',
+        'aria-hidden': true,
```

Comment:
are we sure this should always be hidden?
can you please share accessibility spec for the component, I would ilke to understand the use case better.

Reply:
For media images, yes, but we'll be moving this to a span slot so that users can inject video/etc. as well, where accessibility will be contextual, but default I believe this is correct if it's simply aria-hidden.

I can link our design guidance w/ accessibility spec offline.

Reply:
Migrated to a generic span w/ presentation role, aria-hidden for now, we will be vetting accessibility further as part of stable release.